### ISSUE TITLE
Pytorch model converting

### ISSUE THREAD

**Issue by joshuachris2001** [ggerganov/ggml#18]
is converting from PyTorch possible? similar to TensorFlow conversion.

**Comment by ggerganov** [comment]
Yes, see the [whisper](https://github.com/ggerganov/ggml/blob/master/examples/whisper/convert-pt-to-ggml.py) and [gpt-j](https://github.com/ggerganov/ggml/blob/master/examples/gpt-j/convert-h5-to-ggml.py) examples.
